apps/console/src/features/identity-providers/components/forms/authenticators/email-otp-authenticator-form.tsx

```diff
@@ -115,7 +115,7 @@ interface EmailOTPAuthenticatorFormFieldsInterface {
     /**
      * Allow OTP token to have 0-9 characters only field.
```

Comment:
Let's update the doc comments as well.

Reply:
Resolve with 41555a63bde603bb36b8c7b0c1acf48eea654946